modules/etcd-increase-db.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="etcd-increase-db_{context}"]
+= Increasing the database size for etcd
+
+You can set the disk quota in gigabytes for each etcd instance. If you set a disk quota for your etcd instance, you can specify integer values from 8 to 32. The default value is 8. You can specify only increasing values.
+
+:FeatureName: Increasing the database size for etcd
+include::snippets/technology-preview.adoc[]
+
+[id="etcd-change-db-size_{context}"]
+== Changing the etcd database size
+
+To change the database size for etcd, complete the following steps.
+
+.Procedure
+
+. Check the current value of the disk quota for each etcd instance by entering the following command:
++
+[source,terminal]
+----
+$ oc describe etcd/cluster | grep "Backend Quota"
+----
++
+.Example output
+[source,terminal]
+----
+Backend Quota Gi B: <value>
+----
+
+. Change the value of the disk quota by entering the following command:
++
+[source,terminal]
+----
+$ oc patch etcd/cluster --type=merge -p '{"spec": {"backendQuotaGiB": <value>}}'
+----
++
+.Example output
+[source,terminal]
+----
+etcd.operator.openshift.io/cluster patched
+----
+
+.Verification
+
+. Verify that the new value for the disk quota is set by entering the following command:
++
+[source,terminal]
+----
+$ oc describe etcd/cluster | grep "Backend Quota"
+----
++
+The etcd Operator automatically rolls out the etcd instances with the new values. 
+
+. Verify that the etcd pods are up and running by entering the following command:
++
+[source,terminal]
+----
+
+----
++
+The following output shows the expected entries for master-0. 
++
+.Example output
```

Comment:
@dusk125 In the peer review, the reviewer suggested that we show the completed output instead of in progress output. Can you provide an example of what that might look like?

Reply:
Will do this today, gotta startup a cluster to grab the output

Reply:
@lahinson these are the entries for the full command

`oc get pods -n openshift-etcd`

```
NAME                                                   READY   STATUS      RESTARTS   AGE
etcd-ci-ln-b6kfsw2-72292-mzwbq-master-0                4/4     Running     0          39m
etcd-ci-ln-b6kfsw2-72292-mzwbq-master-1                4/4     Running     0          37m
etcd-ci-ln-b6kfsw2-72292-mzwbq-master-2                4/4     Running     0          41m
etcd-guard-ci-ln-b6kfsw2-72292-mzwbq-master-0          1/1     Running     0          51m
etcd-guard-ci-ln-b6kfsw2-72292-mzwbq-master-1          1/1     Running     0          49m
etcd-guard-ci-ln-b6kfsw2-72292-mzwbq-master-2          1/1     Running     0          54m
installer-5-ci-ln-b6kfsw2-72292-mzwbq-master-1         0/1     Completed   0          51m
installer-7-ci-ln-b6kfsw2-72292-mzwbq-master-0         0/1     Completed   0          46m
installer-7-ci-ln-b6kfsw2-72292-mzwbq-master-1         0/1     Completed   0          44m
installer-7-ci-ln-b6kfsw2-72292-mzwbq-master-2         0/1     Completed   0          49m
installer-8-ci-ln-b6kfsw2-72292-mzwbq-master-0         0/1     Completed   0          40m
installer-8-ci-ln-b6kfsw2-72292-mzwbq-master-1         0/1     Completed   0          38m
installer-8-ci-ln-b6kfsw2-72292-mzwbq-master-2         0/1     Completed   0          42m
revision-pruner-7-ci-ln-b6kfsw2-72292-mzwbq-master-0   0/1     Completed   0          43m
revision-pruner-7-ci-ln-b6kfsw2-72292-mzwbq-master-1   0/1     Completed   0          43m
revision-pruner-7-ci-ln-b6kfsw2-72292-mzwbq-master-2   0/1     Completed   0          43m
revision-pruner-8-ci-ln-b6kfsw2-72292-mzwbq-master-0   0/1     Completed   0          42m
revision-pruner-8-ci-ln-b6kfsw2-72292-mzwbq-master-1   0/1     Completed   0          42m
revision-pruner-8-ci-ln-b6kfsw2-72292-mzwbq-master-2   0/1     Completed   0          42m
```